CONTRIBUTING.md

```diff
@@ -5,12 +5,13 @@ please read the [code of conduct](CODE_OF_CONDUCT.md).
 
 ## Setup
 
-> Install yarn on your system: https://yarnpkg.com/en/docs/install
-
 ```sh
-$ git clone https://github.com/yarnpkg/yarn.git
+$ # Install yarn on your system: https://yarnpkg.com/en/docs/install
+$ # Fork the repo: https://github.com/yarnpkg/yarn
```

Comment:
Same thing for this instruction, I don't think it should be in a comment of bash script.